packages/kbn-es/src/cluster.js

```diff
@@ -259,6 +259,9 @@ exports.Cluster = class Cluster {
     this._process = execa(ES_BIN, args, {
       cwd: installPath,
       stdio: ['ignore', 'pipe', 'pipe'],
+      env: {
+        [options.bundledJDK && 'JAVA_HOME']: '',
```

Comment:
Would you mind doing this condition outside of the object? setting the `false` environment variable to `''` feels a little iffy. Same for below.